truffle-seq/OwnerManagedLite_test_activate_deactivate_dapp.md

```diff
@@ -0,0 +1,775 @@
+
+
+```plantuml
+
+
+@startuml
+
+autonumber
+skinparam legendBackgroundColor #FEFECE
+
+<style>
+      header {
+        HorizontalAlignment left
+        FontColor purple
+        FontSize 14
+        Padding 10
+      }
+    </style>
+
+header Insights by Truffle
+
+title Txn Hash: 0x9382cea9c31db99cc01558ffe2c0846e7e4de799c5a4fb2f1ac9d4bcbab12d01
+
+
+actor EOA as "EOA"
+participant OwnerManagedLite_01 as "OwnerManagedLite_01"
+
+"EOA" -> "OwnerManagedLite_01" ++: AddState(\n\
+<#FEFECE,#FEFECE>|= type |= name |= value |\n\
+| address | ownerAddress | 0x26dF443a80B4281eD90237EfAEB788F7096F0a67 |\n\
+| bool | isActive | true |\n\
+) { 0.0 GWEI }
+"OwnerManagedLite_01" -> "EOA" --: 
+
+legend
+Participant details
+<#FEFECE,#D0D000>|= Alias |= Contract name |= Address |
+<#FEFECE>| EOA | Externally Owned Account | 0x26dF443a80B4281eD90237EfAEB788F7096F0a67 |
+<#FEFECE>| OwnerManagedLite_01 | OwnerManagedLite | 0x6ee441E54faA03419F3b64cA48668A4f02EB052F |
+endlegend
+
+@enduml
+```
+
+##### d3, tx: 0x146f3d061979cebe881ff14f1d676d09c1c184ae837979eddf23a36e72bd896d
+
+[SVG :telescope:](https://www.planttext.com/api/plantuml/svg/xLLTQzim57tNhn3TIqTN9jcoP9Skr4bjEjX8MVfO68frdPWwIf3bBkco_xuRnmij2tieVPmnklBzb8vqZmcvQhnslbtKXEZMBssxc88ZpKDbLzhf1QrX1jOCTV4mSylMcj6oNZhwBi_oR9GHSjduJGq3Ghjd3jg0ep_xJqflbgvwMbglwxIkPdO1rcF6qXySSZJkKwvQjwhXXM5IFG7bug2xrSPKTaOvwrM_EddviLy4wUj_jaqrc_k6JZVqphLbYNc9htmDz6vjwOrkvXUKhRcGPMYOv8bA2fX27FEovABaHYffM5BmWiT2Gnmgz01ZoY3KeGGLJ4sSI4CGiiBZqhDnIdL3Jr2U48JDLqMrqhZLyGyBxgksUWRcI-NXE-ETun7z2I5TF3qV7BVJixCBQW0hLe_QmyJZSFgU7FVzPn873VZHNDiPZEqhyvoVNz1lu5jdwUczlIUN_OtuqCl1zXFrcnLGb5OlElcewnOdE_SjRIl4PqlnLL6W91TC9Z9CE5T2H2naKOmgaKHAXZpXJ4eXao28E9gOY1MB4XKmBWEfK273M8WmK4hkqxyAbeL-WB6R_87cTTWMNUp_mtdBmzbjvurZzfssyo5xWYEtpzhMWDTLtJp3xPfTCyPsk26FOGVZP8G4vR0R3mYcnk3XeUqGDzXsr83ZseEpkgut7IKOcXO5akaEOxOEfCaH81speGXY3jS92qALbMasL76SAvR8d6cfNYO-HYr_Qxi24a08daMYr2aB1K_oS2f5aOfOoZWL9Gko8Ok270iW4ZqUv0f7_2ty1W00)
+
+
+```plantuml
+
+
+@startuml
+
+autonumber
+skinparam legendBackgroundColor #FEFECE
+
+<style>
+      header {
+        HorizontalAlignment left
+        FontColor purple
+        FontSize 14
+        Padding 10
+      }
+    </style>
+
+header Insights by Truffle
+
+title Txn Hash: 0x146f3d061979cebe881ff14f1d676d09c1c184ae837979eddf23a36e72bd896d
+
+
+actor EOA as "EOA"
+participant OwnerManagedLite_01 as "OwnerManagedLite_01"
+
+"EOA" -> "OwnerManagedLite_01" ++: deactivateState()
+"OwnerManagedLite_01" -> "OwnerManagedLite_01" ++: resetChangeOnState()
```

Comment:
ooh nice! Can you share some more details?

Reply:
It looks like all my calls do a reset at the end of the state change, so I could either remove from the end or the beginning. I just have been over eager to make sure it got reset to avoid conflicts.